### DISCUSSION TITLE
Form.List 内部的 Form.Item 的 `namePath` 有办法不受 Form.List 的影响么

### DISCUSSION THREAD

**Discussion by OneMoreJack** [ant-design/ant-design#40210] [Q&A]
如果 Form.Item 嵌套在 Form.List 内部，其 namePath 最终会被处理成 `[...Form.List 的 namePath, ...Form.Item 的 namePath]`;
有什么方式可以让 Form.Item 的 namePath 不受 Form.List 的影响么?  如：`[...Form.Item 的 namePath]`

**Comment by zombieJ** [comment]
在 List 下的目的就是关联路径，如果不需要外面不用包 List 即可。

**Reply by OneMoreJack** [reply]
好的，谢谢。
